### PR TITLE
Choose how to formalise repayment screen

### PR DESCRIPTION
**Pull request by RosDiaf**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/modules/i18n/locales/cy.json [hunk 1]
```diff
@@ -640,6 +640,18 @@
       "PAGE_TITLE": "Your date of birth",
       "TITLE": "What is your date of birth?"
     },
+    "CLAIMANT_FORMALISE_REPAYMENT_TYPE": {
+      "PAGE_TITLE": "Choose how to formalise repayment",
+      "SIGN_A_SETTLEMENT_AGREEMENT": {
+     	  "LABEL": "Sign a settlement agreement",
```
Comment (GorilaNaranja) [inline src/main/modules/i18n/locales/cy.json:646]: alphabetic order

**Review by TatsyMaslova — APPROVED** [review]

File: src/main/modules/nunjucks/index.ts [hunk 2]
```diff
@@ -26,6 +26,7 @@ import {InterestClaimFromType} from '../../common/form/models/claimDetails';
 import {InterestEndDateType} from '../../common/form/models/claimDetails';
 import * as urls from '../../routes/urls';
 import {InterestClaimOptionsType} from '../../common/form/models/claim/interest/interestClaimOptionsType';
+import {ChooseHowToProceed} from 'common/models/chooseHowToProceed';
```
Comment (GorilaNaranja) [inline src/main/modules/nunjucks/index.ts:29]: Do we need access to path like that? I'm not sure, just asking. Same question for all imports.
`'../../common/models/chooseHowToProceed'`

**Review by laarmada — APPROVED** [review]

**Review by GorilaNaranja — APPROVED** [review]

File: src/main/modules/i18n/locales/cy.json (hunk 1 above)
Comment (RosDiaf) [reply src/main/modules/i18n/locales/cy.json]: Added

File: src/main/modules/nunjucks/index.ts (hunk 2 above)
Comment (RosDiaf) [reply src/main/modules/nunjucks/index.ts:29]: Relative path removed

File: src/main/modules/i18n/locales/cy.json [hunk 3]
```diff
@@ -640,6 +640,18 @@
       "PAGE_TITLE": "Your date of birth",
       "TITLE": "What is your date of birth?"
     },
+    "CLAIMANT_FORMALISE_REPAYMENT_TYPE": {
```
[the remark this answers is not in the document]
Comment (RosDiaf) [reply src/main/modules/i18n/locales/cy.json:643]: I believe the translation will be added on a different ticket.

**Pull request closed without merging**